docs/pages/concepts/session-strategies.mdx

```diff
@@ -9,7 +9,7 @@ When a user visits your application, you want them to not have to log in every s
   evaluate based on the requirements of your application.
 </Callout>
 
-You can configure the session strategy using the [`session.strategy`](/reference/core#session) option in the main Auth.js config file.
+You can configure the session strategy using the [`session.strategy`](/reference/core#stategy) option in the main Auth.js config file.
```

Comment:
Did you mean `#strategy`?

Reply:
Oops! That's embarrassing. Thank you for catching it.

I had to recreate this PR after rebasing my fork of `next-auth` and then syncing it with `next-auth` deleted the commit.